Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -4,34 +4,281 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. _k8s_install_quick:
 .. _k8s_quick_install:
+.. _k8s_install_standard:
 
 ******************
 Quick Installation
 ******************
 
-This guides takes you through the quick installation procedure.  The default
-settings will store all required state using Kubernetes custom resource
-definitions (CRDs). This is the simplest installation method as it only depends
-on Kubernetes and does not require additional external dependencies. It is a
-good option for environments up to about 250 nodes. For bigger environments or
-for environments which want to leverage the clustermesh functionality, a
-kvstore set up is required which can be set up using an :ref:`k8s_install_etcd`.
+This guide will walk you through the quick default installation. It will
+automatically detect and use the best configuration possible for the Kubernetes
+distribution you are using. All state is stored using Kubernetes CRDs.
+
+This is the best installation method for most use cases.  For large
+environments (> 500 nodes) or if you want to run specific datapath modes, refer
+to the :ref:`k8s_install_advanced` guide.
 
 Should you encounter any issues during the installation, please refer to the
 :ref:`troubleshooting_k8s` section and / or seek help on the `Slack channel`.
 
-Please consult the Kubernetes :ref:`k8s_requirements` for information on  how
-you need to configure your Kubernetes cluster to operate with Cilium.
+Install the Cilium CLI
+======================
+
+.. include:: install-cli.rst
+
+Create the Cluster
+===================
+
+If you don't have a Kubernetes Cluster yet, you can use the instructions below
+to create a Kubernetes cluster locally or using a managed Kubernetes service:
+
+.. tabs::
+
+    .. group-tab:: GKE
```

Comment:
[Nit] For consistency, we should rename tabs for GKE/AKS/EKS to include the platform names, as in the next set of tabs.

Reply:
Unlike the tabs below, these are actually specific to GKE, EKS, AKS, and so on.

Reply:
Sorry, I'm not sure I follow 🤔
Does that mean there is a difference between the GKE tab highlighted above and the GCP/GKE tab present in the next set of tabs?
```rst
.. group-tab:: GCP/GKE
```

Reply:
Yes, the "GKE" tab in create cluster is creating a GKE cluster. The "GCP/GKE" tab in "Install Cilium" will function for both GKE and self-managed clusters in GCP. I'm OK to also label GCP above if you think that's less confusing overall.

Reply:
Ah I see! I think I got mostly confused by the fact the `GCP/GKE`-like tabs explicitly only mention provider-managed clusters (e.g. `Install Cilium into the GKE cluster:`), so I did not think the docs were targeting self-managed GCP clusters.

Perhaps we ought to rephrase some parts a bit differently for self-managed clusters.